spec/controllers/tagged_content_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe TaggedContentController, type: :controller do
+  include ContentSchemaHelpers
+  include ServicesRequestHelpers
+
+  describe "GET #show" do
+    before(:each) do
+      @example_schema = example_content_schema("taxon", "taxon_with_child_taxons")
+      @path = "foo/bar"
+      @document_type = "publication"
+      stub_content_store("/#{@path}", @example_schema)
+      stub_rummage_with_doc_type(@example_schema["content_id"], @document_type)
```

Comment:
You should be able to use test helper `stub_any_rummager_search` from gds-api-adapters as you aren't dependent on the results of the rummager search in this test.

Reply:
The same is true for stubbing content-store. You should be able to use `content_store_has_item("/#{@path}")`

Reply:
The reason we wrote these test helpers were due to the fact that some of the external stub methods weren't working due to some WebMock dependancy problems, but if you have used them on this project then I'm will give it a go.

Reply:
Removing them works fine. I have also refactored the code to remove any redundant mocking that can be done with API Adaptors